vulcano/command/models_test.py

```diff
@@ -19,53 +21,77 @@ def sample_command(arg1, arg2=None, arg3='Hello'):
+    return  # pragma: no cover
 
 
 class MyTestCase(unittest.TestCase):
-    @patch('vulcano.command.models.get_func_inspect_result')
+    @patch("vulcano.command.models.get_func_inspect_result")
     def test_assert_should_use_args_if_present(self, gfunc_inspect_mock):
-        description = 'Description'
-        name = 'Name'
-        command = models.Command(func=sample_command, description=description, name=name)
-        self.assertEqual(command.short_description, description, 'Should use description from args whenever possible')
-        self.assertEqual(command.name, name, 'Should use name from args whenever possible')
+        description = "Description"
+        name = "Name"
+        command = models.Command(
+            func=sample_command, description=description, name=name, args_opts=ARGS_OPTS
+        )
+        self.assertEqual(
+            command.short_description,
+            description,
+            "Should use description from args whenever possible",
+        )
+        self.assertEqual(
+            command.name, name, "Should use name from args whenever possible"
+        )
         self.assertEqual(command.func, sample_command)
         self.assertTrue(command.visible)
         gfunc_inspect_mock.assert_called_with(sample_command)
 
     def test_help(self):
-        command = models.Command(sample_command)
-        self.assertEqual(command.help, 'sample_command: \tHere goes the short description\nHere goes the long description\n\t Args:\n\t\t*arg1(str): Here goes the argument 1 desc \n\t\targ2(str): Here goes the argument 2 desc \n\t\targ3(default: Hello): None\n')
+        command = models.Command(sample_command, args_opts=ARGS_OPTS)
+        self.assertEqual(
+            command.help,
+            "sample_command: \tHere goes the short description\nHere goes the long description\n\t Args:\n\t\t*arg1(str): Here goes the argument 1 desc \n\t\targ2(str): Here goes the argument 2 desc \n\t\targ3(default: Hello): "
+            + str(ARGS_OPTS["arg3"])
+            + "\n",
+        )
 
     def test_visible_should_accept_a_function(self):
         visible_func = MagicMock()
         visible_func.visible.side_effect = [True, False]
-        command = models.Command(sample_command, show_if=visible_func.visible)
+        command = models.Command(
+            sample_command, show_if=visible_func.visible, args_opts=ARGS_OPTS
+        )
         self.assertTrue(command.visible)
         self.assertFalse(command.visible)
         visible_func.visible.assert_has_calls([call(), call()])
 
     def test_command_completer(self):
-        command = models.Command(sample_command)
-        completer = command.command_completer
+        command = models.Command(sample_command, args_opts=ARGS_OPTS)
+        completer = command.command_completer[0]
         self.assertIsInstance(completer, tuple)
-        name, description = command.command_completer
+        name, description = command.command_completer[0]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Value 'command.command_completer' is unsubscriptable](https://app.codacy.com/manual/david-gp87/vulcano/pullRequest?prid=5134582)